config/locales/proper.en.yml

```diff
@@ -53,6 +53,13 @@ en:
       accordion:
         header: "Browse Service Catalog"
       back_to_top: "Back to Top"
+      short_interaction: "Short Interaction"
```

Comment:
This value is pulled from `t(:proper)[:right_navigation][:short_interaction][:header]` in the application

https://github.com/sparc-request/sparc-request/blob/v3.5.0/app/views/service_requests/right_navigation/_right_navigation.html.haml